tests/integration_tests/test_parameter_example.py

```diff
@@ -188,7 +188,7 @@ def create_surface(self, file_name: str) -> None:
         ).to_file(file_name, fformat="irap_ascii")
 
 
-@settings(max_examples=10)
+@settings(max_examples=100000)
```

Comment:
Why?

Reply:
Testing until it fails. This doesn't work now that there is a global timeout and I spent too much time debugging it. :facepalm: